config/rbac/role.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   creationTimestamp: null
-  name: controller
+  name: manager-role
```

Comment:
If you change the name of the ClusterRole, you also change ClusterRoleBinding.
BTW, why did you change it? By kubebuilder?

Reply:
Sorry, I'll fix it. 
> By kubebuilder?

Yes.

Reply:
fixed it.
https://github.com/topolvm/pvc-autoresizer/pull/41/commits/250ebe678a3ebe4842c90c20da6f0f3d97abaa86

Reply:
Note: https://github.com/kubernetes-sigs/controller-tools/issues/513#issuecomment-722219300

Reply:
Scaffolding code is here: https://github.com/kubernetes-sigs/kubebuilder/blob/v3.1.0/pkg/plugins/golang/v2/scaffolds/internal/templates/config/rbac/role_binding.go#L32-L55